app/schemas/email.py

```
@@ -0,0 +1,162 @@
+    content_base64: str = Field(
+        ...,
+        min_length=1,
+        max_length=MAX_ATTACHMENT_BASE64_CHARS,
+        description="Base64-encoded attachment content",
+    )
+
+    @field_validator("filename")
+    @classmethod
+    def _validate_filename(cls, value: str) -> str:
+        if value in {".", ".."}:
+            raise ValueError("filename must not be a path traversal token")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)